### ISSUE TITLE
Less than before dollar sign: <$

### ISSUE THREAD

**Issue by a-pav** [tonsky/FiraCode#1469]
Less than before dollar sign doesn't look how it should.

![fira-code-issue](https://user-images.githubusercontent.com/14939129/190085759-348e459d-5406-43e1-81d6-708c1de86ed4.png)

This was on VSCodium with below settings:

```
"editor.fontFamily": "'Fira Code'",
"editor.fontLigatures": "'zero', 'cv02', 'cv15', 'cv30', 'ss01', 'ss02', 'ss03', 'ss04', 'ss05', 'ss08', 'calt'",
```

**Comment by wopian** [comment]
That looks correct to me 🤔

**Comment by tonsky** [comment]
Yeah what’s wrong?

**Comment by tonsky** [comment]
I think those come from Haskell? I’m not sure

**Comment by wopian** [comment]
Yes https://hackage.haskell.org/package/base-4.8.2.0/docs/Data-Functor.html#v:-60--36--62-

(`<$`, `<$>` and `$>`)

**Comment by tonsky** [comment]
Yes, they make a good case for stylistic set
